Adds nest sustenance status effect, halves xenomorph nest escape time

Rhials opened the pull request:
## About The Pull Request

Hosts who are buckled to a xenomorph nests will gestate their xenomorph egg 20% faster (4 minutes in a nest, 5 without) as long as they are trapped. To keep them alive in the nest, they will recieve temperature stabilization and be healed until they are pulled (or break) free. 

To ensure the gift of life is not wasted by captured humans, they can now break free from resin nests after one minute of struggling, down from two.
## Why It's Good For The Game

There's currently no incentive to not kill facehugger infectees immediately after they're impregnated. Dead bodies can't run away, gestate at the same speed, and hives are often breached from space making them deadly to live captures regardless. This serves the dual purpose of incentivizing live captures, while also making them a feasible option, without removing the ability to infect dead bodies and seriously impacting the xenomorph playstyle.

Cutting the escape timer in half (while also ensuring that captured humans are alive enough to break out) means there will be many more opportunities for live captures to escape. Additionally, the xeno players will have to spare manpower to actively defend hosts and if they wish to ensure they do not escape. Hopefully it should give more reason to play defensively and set up a proper hive, instead of the hypermobile-hyperviolent playstyle you sometimes encounter.
## Changelog
:cl: Rhials
balance: Xenomorph nests will now sustain humans that are buckled to them, and gestate xenomorph eggs slightly faster.
balance: Xenomorph nests evolved healing capabilities in exchange for some of their grip. They now take only one minute to escape from.
/:cl:

Rhials committed:
lets try

Rhials committed:
lets try

Rhials committed:
further progress

Rhials committed:
further progress

Rhials committed:
screen alert icon

status effect is only applied to living targets with a facehugger (before impregnation) or with the xeno host trait (after impregnation)

Reduces how quickly the sustenance speeds up the infection because it kills you before you can even break out with how fast it is

Rhials committed:
screen alert icon

status effect is only applied to living targets with a facehugger (before impregnation) or with the xeno host trait (after impregnation)

Reduces how quickly the sustenance speeds up the infection because it kills you before you can even break out with how fast it is

Rhials committed:
fixes a problem

if you put a facehugger on a nested person they now get the status effect

Rhials committed:
fixes a problem

if you put a facehugger on a nested person they now get the status effect

Rhials committed:
too fast

Rhials committed:
too fast

Rhials committed:
better ways of doing some things

Rhials committed:
better ways of doing some things

Rhials committed:
status effect stops when you die

Rhials committed:
status effect stops when you die

Rhials committed:
granar

Rhials committed:
granar

Rhials committed:
whoops

Rhials committed:
whoops

Rhials committed:
actual heals

Rhials committed:
actual heals

Rhials committed:
a really scornlike PR

Rhials committed:
a really scornlike PR

Serxule commented:
i dont think nest should keep healing, but be capped to a certain percentage

Serxule commented:
oh also it should be kept in mind, people can just be killed after being infected, they dont need to be alive so a 1 minute decrease probably isnt worth it

Rhials committed:
Merge remote-tracking branch 'upstream/master' into fresh

Rhials committed:
Merge branch 'fresh' of https://github.com/Rhials/tgstation into fresh

Rhials committed:
readds sprite

LemonInTheDark commented on code/game/objects/structures/beds_chairs/alien_nest.dm:
```diff
@@ -44,6 +44,8 @@
 					span_notice("You break free from the gelatinous resin!"),\
 					span_hear("You hear squelching..."))
 
+			M.remove_status_effect(/datum/status_effect/nest_sustenance)
```
what if you get off via other methods?

LemonInTheDark replied:
see what happens if you buckle to the thing, and then it gets deleted while you're on it. if it works as you want that's enough

LemonInTheDark commented on code/game/objects/structures/beds_chairs/alien_nest.dm:
```diff
@@ -32,9 +32,9 @@
+					span_notice("You struggle to break free from the gelatinous resin... (Stay still for one minute.)"),\
 					span_hear("You hear squelching..."))
-				if(!do_after(M, 1200, target = src))
+				if(!do_after(M, 60 SECONDS, target = src))
```
I'm not sure how to feel about this, 60 seconds seems real quick, especially since the gestation would still be twice the old delay. I do like the whole incentivization of keeping people alive to pop tho, that vibes

Rhials committed:
lets try  this

also fixes an accidental thing I did that would make the status effect delete itself oops
